front/app/components/UI/QuillEditor/altModule.tsx

```diff
@@ -0,0 +1,96 @@
+import Quill, { QuillOptionsStatic } from 'quill';
+
```

Comment:
This is our new module + image blot implementation that allows us to add the input for the alt text to all images. It's heavily inspired by this custom modules that allow to add captions in a similar way: https://gist.github.com/tranduongms1/584d43ec7d8ddeab458f087adbeef950